docs/VLSI/Tutorial.rst

```diff
@@ -107,6 +105,16 @@ This contains the Hammer configuration for this example project. Example clock c
 
 First, set ``technology.asap7.tarball_dir`` to the absolute path of where the downloaded the ASAP7 PDK tarball lives.
 
+SRAM Compilation
+^^^^^^^^^^^^^^^^
+Hammer can automatically add the collateral for the SRAMs in the ``Sha3RocketConfig``.
```

Comment:
Can we add another sentence or two? What collateral? Which SRAMs? This sentence lacks a lot of context- even I don't really know what it means.

Reply:
I agree with John. I think you mean gds, lib, lef for the SRAMs, but maybe you mean the hammer json?